### ISSUE TITLE
No matching distribution found for jsmin==2.2.2

### ISSUE THREAD

**Issue by NgHuuAn** [ngoduykhanh/PowerDNS-Admin#1004]
Hi, 

We encountered the following error while installing the required applications.

OS Version:   CentOS Linux release 7.9.2009 (Core)
Ker ver:  3.10.0-1160.42.2.el7.x86_64

```
    ERROR: Command errored out with exit status 1:
     command: /opt/web/powerdns-admin/flask/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-install-vxumb7y1/jsmin_e626aecddcd743489bbea9922bdf4135/setup.py'"'"'; __file__='"'"'/tmp/pip-install-vxumb7y1/jsmin_e626aecddcd743489bbea9922bdf4135/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-5ur3vvl_
         cwd: /tmp/pip-install-vxumb7y1/jsmin_e626aecddcd743489bbea9922bdf4135/
    Complete output (1 lines):
    error in jsmin setup command: use_2to3 is invalid.
    ----------------------------------------
WARNING: Discarding https://files.pythonhosted.org/packages/17/73/615d1267a82ed26cd7c124108c3c61169d8e40c36d393883eaee3a561852/jsmin-2.2.2.tar.gz#sha256=b6df99b2cd1c75d9d342e4335b535789b8da9107ec748212706ef7bbe5c2553b (from https://pypi.org/simple/jsmin/). Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
ERROR: Could not find a version that satisfies the requirement jsmin==2.2.2 (from versions: 2.0, 2.0.1, 2.0.2, 2.0.2.post1, 2.0.3, 2.0.5, 2.0.6, 2.0.7, 2.0.8, 2.0.9, 2.0.10, 2.0.11, 2.1.0, 2.1.1, 2.1.2, 2.1.3, 2.1.4, 2.1.5, 2.1.6, 2.2.0, 2.2.1, 2.2.2, 3.0.0)
ERROR: No matching distribution found for jsmin==2.2.2
```
Please help.

Thanks !

**Comment by NgHuuAn** [comment]
Is it possible to temporarily fix it by changing the version of jsmin by editing the requirements.txt file, but is there any problem with the operation?

**Comment by ngoduykhanh** [comment]
Fix by #1021
